### ISSUE TITLE
Bump net-mail/offlineimap3 to Python 3.10

### ISSUE THREAD

**Issue by r7l** [ferki/gentoo-overlay#2]
Hello, just recently Gentoo made Python 3.10 default and from what it seems, offlineimap and it's dependencies support this version already. Would it be possible to add 3.10 to the Ebuilds?

**Comment by ferki** [comment]
Thanks for opening the issue about it, @r7l!

I did not get a GitHub notification about this issue for some reason. I made sure I'm now a watcher of all activities in this repo :upside_down_face:

Incidentally, on the next day I already bumped Python compatibility for net-mail/offlineimap3 and all its dependencies in this overlay (dev-python/imaplib2 and dev-python/rfc6555) on [these commits](https://github.com/ferki/gentoo-overlay/compare/ferki:1d49de6%5E%5E%5E...ferki:1d49de6).

It works for me with dev-lang/python-3.10.5 since then, but let me know if you still see something odd.
